docs/Reference/Cakeshop/Overview.md

```diff
@@ -0,0 +1,79 @@
+# Getting started with Cakeshop
+
+## Download
+
+Binary packages are available on the [GitHub releases page](https://github.com/ConsenSys/cakeshop/releases).
+
+## Configuration
+
+Cakeshop is a Spring Boot application, so you may place an `application.properties` file in the working directory to override any default configuration values. For more info, see the [configuration page](Configuration.md).
```

Comment:
Better for search engine: use documentation instead of page which is a too generic term
```suggestion
Cakeshop is a Spring Boot application, so you may place an `application.properties` file in the working directory to override any default configuration values.
For more info, see the [configuration documentation](Configuration.md).
```